src/workspace/js/solutions/task2.js

```diff
@@ -0,0 +1,2 @@
+let message = 'Hello World';
```

Comment:
```suggestion
const message = 'Hello World';
```
As the string is not being reassigned I think that const should be used instead of let